go/client/cmd_chat_api.go

```diff
@@ -402,20 +389,20 @@ func (c *CmdChatAPI) sendV1(ctx context.Context, arg sendArgV1) Reply {
 			return c.errReply(err)
 		}
 		rlimits = append(rlimits, ncres.RateLimits...)
-		conversation = ncres.Conv.Info
+		convTriple, convID = ncres.Conv.Info.Triple, ncres.Conv.Info.Id
 	case 1:
-		conversation = existing[0].Info
+		convTriple, convID = existing[0].Metadata.IdTriple, existing[0].Metadata.ConversationID
 	default:
 		return c.errReply(fmt.Errorf("multiple conversations matched"))
 	}
 
 	postArg := chat1.PostLocalArg{
-		ConversationID: conversation.Id,
+		ConversationID: convID,
 		MessagePlaintext: chat1.NewMessagePlaintextWithV1(chat1.MessagePlaintextV1{
 			ClientHeader: chat1.MessageClientHeader{
-				Conv:        conversation.Triple,
-				TlfName:     conversation.TlfName,
-				TlfPublic:   conversation.Visibility == chat1.TLFVisibility_PUBLIC,
+				Conv:        convTriple,
+				TlfName:     *arg.convQuery.TlfName,
```

Comment:
Did you mean conversation ID is used anyway? Yes, but that's used for selecting conversation on server. However, we are still using the TLF name to get the keys, so even if it's the wrong conversation on server, we wouldn't be encrypting for wrong TLF